docs/assessment-update.md

```diff
@@ -0,0 +1,47 @@
+---
+# SPDX-License-Identifier: CC0-1.0
+# SPDX-FileCopyrightText: SPDX-FileCopyrightText: 2025 Standard for Public Code Authors, https://www.standardforpubliccode.org/AUTHORS; 2022-2024 The Foundation for Public Code <info@publiccode.net>
+type: Index
+---
+
+# Assessment Update
+
+The commands `git diff` and `patch` can be used to to reduce the effort required to update an existing assessment of a codebase to a newer version of the Standard for Public Code.
+
+## Create the initial patch
+
+From a clone of the [Standard repository](https://github.com/standard-for-public-code/standard-for-public-code/), prepare the patch:
+
+```bash
+git diff 0.8.0 0.8.1 docs/review-template.html > review-template-0.8.0-0.8.1.patch
+```
+
+## Prepare the patch
+
+The patch lines at the top of the path reflect the template file path:
+
+```diff
+--- a/docs/review-template.html
++++ b/docs/review-template.html
+```
+
+Edit those lines to match the path to the assessment in the repository you wish to update, for example:
+
+```diff
+--- a/path/to/standard-for-public-code-assessment.html
++++ b/path/to/standard-for-public-code-assessment.html
+```
+
+## Apply the patch
+
+From a clone of the assessed repository, apply the modified patch:
+
+```bash
+patch -Np1 -i /path/to/modified-review-template.patch
```

Comment:
Shall we avoid usage of shorthand flags for this command?

```bash
patch -Np1 -i /path/to/modified-review-template.patch
```

`patch --help`

`-N  --forward  Ignore patches that appear to be reversed or already applied.`
`-p NUM  --strip=NUM  Strip NUM leading components from file names.`
`-i PATCHFILE  --input=PATCHFILE  Read patch from PATCHFILE instead of stdin.`

Would this command be equivalent?

```bash
patch --forward -strip=1 --input=/path/to/modified-review-template.patch
```

Reply:
Yes, very good idea. I shall adjust it for long-options. It might be a couple of days.